notebooks/mg-20230520-00-binary-classifier.ipynb

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        .where("rank &lt;= 250")
</pre><p>Since this is a call/no-call classifier, I think you can drop this line here and just filter anything with a probability above 0.5 (or some other reasonable threshold).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>